### ISSUE TITLE
bug: ScreenReader iOS does not work with swipe gestures on ion-select >= 8.3.4

### ISSUE THREAD

**Issue by mopat** [ionic-team/ionic-framework#30043]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/docs/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

v8.x

### Current Behavior

In version 8.3.4, the behavior of the screen reader (VoiceOver) has changed when swiping right. The buttons (e.g., Cancel/OK) are no longer accessible using the right-swipe gesture.
Additionally, the alert is not properly focused when the alert opens.

### Expected Behavior

- When swiping right using the screen reader (VoiceOver), all actionable buttons (e.g., Cancel, OK) should be reachable via the gesture.
- The alert should automatically receive proper focus, ensuring that it is announced and interactable immediately upon appearing.
- The user should be able to navigate through the alert content and buttons seamlessly using VoiceOver gestures.

Check version 8.3.3 where it used to work this way.

### Steps to Reproduce

- Open the application and navigate to a screen that contains an ion-select component.
- Activate the ion-select to display its options (e.g., by tapping on it).
- Observe the alert/modal that appears, listing the selectable options.
- Enable VoiceOver on the device.
- Attempt to swipe right to navigate through the options or actionable buttons (e.g., Cancel/OK).

Notice whether:
- The buttons (e.g., Cancel/OK) are reachable using the right-swipe gesture.
- The alert/modal is properly focused when it appears.

### Code Reproduction URL

https://ionicframework.com/docs/api/select

### Ionic Info

Ionic:

Ionic CLI: 7.2.0
Ionic Framework: @ionic/angular 8.3.4
@angular-devkit/build-angular: 18.2.11
@angular-devkit/schematics: 18.2.11
@angular/cli: 18.2.11
@ionic/angular-toolkit: 12.1.1
Capacitor:

Capacitor CLI: not installed
@capacitor/android: 6.1.2
@capacitor/core: 6.1.2
@capacitor/ios: 6.1.2
Cordova:

Cordova CLI: not installed
Cordova Platforms: not available
Cordova Plugins: not available
Utility:

cordova-res: 0.15.4
native-run: not installed globally
System:

ios-deploy: 1.12.2
NodeJS: v20.17.0
npm: 10.8.2
OS: macOS Unknown
Xcode: Xcode 16.1 Build version 16B40

**Comment by svzi** [comment]
We experience the same issue in one of our projects after the recent update! We needed to switch back to 8.3.3 for that reason.
